account_check_printing_report_base/tests/test_check_printing_report.py

```diff
@@ -135,4 +135,4 @@ def test_check_printing_with_layout(self):
             payment.print_checks()
         except UserError as e:
             pass
-        self.assertEquals(e, False)
+        self.assertEqual(e, False)
```

Comment:
Why changing this?

Reply:
Well, it's changed when you run ```2to3 -wnj4 --no-diffs . ``` as suggested here:
https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-11.0

Reply:
Yeah, but you must run it not on the entire repository, but on your module folder.

Reply:
![seleccion_001](https://user-images.githubusercontent.com/7165771/37893152-750d6ec4-30da-11e8-834f-f5588cb8dda6.png)

Reply:
You are right, it was a mistake, I'll fix it